web_courier.py

```diff
@@ -11,16 +17,49 @@ class WebCourier:
     The courier will then make the async database transaction with the correct model
     '''
 
-    baseURL = 'https://google.com/search?q='
-
-    def __init__(self, loop, url, cloud_generator):
-        self.url = url
+    def __init__(self, loop, cloud_generator):
         self.loop = loop
         self.session = aiohttp.ClientSession()
         self.cloud_generator = cloud_generator
 
-    async def __call__(self):
-        pass
+
+    async def _fetch(self, url):
+        async with self.session.get(url) as resp:
+            status = resp.status
+            logging.info(f"Received status {status} from Google API.")
+            assert status == 200
+            return await resp.json()
+
+
+    async def __call__(self, url):
+        task = [self._fetch(url)]
+        done, pending = await asyncio.wait(
+            task,
+            loop=self.loop,
+            return_when=ALL_COMPLETED
+        )
+        for task in done:
+            data = task.result()
+            for article in data['articles']:
+                content = article['description']
```

Comment:
I'm not sure if using the description here is going to be the best method, but thought it would work for now. There's also a 'content' field returned in the json too. Do you think we should effectively just use the title or try to analyze the whole article using 'content'?